toolhub/demo/domain_generator.py

```diff
@@ -0,0 +1,44 @@
+import click
+
+import openai
+
+from toolhub.demo import openai_assistant
+from toolhub.lib import auth
+from toolhub.lib import registry
+from toolhub.integrations.openapi import provider as openapi_provider
+from toolhub.integrations.rapidapi import provider as rapidapi_provider
+
+
+_OPENAI_KEY = ''  # replace with your OpenAI API key"
```

Comment:
`demo/utils.py`'s `auth_ctx()` and `openai_client()` solve this cleanly by allowing settings files, i.e. not forcing hard-coded keys (and all the git trickery that comes with not accidentally committing them).

Please use those instead.

Reply:
I would like to make the examples as easy to use as possible, and not optimize for our own ease. This is much easier for someone using the codebase for the first time.